web/packages/teleterm/src/services/tshd/types.ts

```diff
@@ -251,6 +251,12 @@ export type TshClient = {
     rootClusterUri: uri.RootClusterUri,
     abortSignal: TshAbortSignal
   ) => Promise<WaitForConnectMyComputerNodeJoinResponse>;
+  deleteConnectMyComputerNode: (
+    clusterUri: uri.RootClusterUri
+  ) => Promise<void>;
+  getConnectMyComputerNodeName: (
+    uri: uri.RootClusterUri
+  ) => Promise<uri.ServerUri>;
```

Comment:
Shouldn't it be `Promise<string>`? This RPC doesn't return the URI, it returns just the UUID alone.